ca/django_ca/openssh/__init__.py

```diff
@@ -0,0 +1 @@
+from .extensions import *
```

Comment:
Please no wildcard imports (flake8 rightfully complains about these). Also name exported extensions in `__all__`, as mypy will otherwise complain.

Reply:
> Also name exported extensions in __all__, as mypy will otherwise complain.

```python
__all__ = (
    'SshHostCaType',
    'SshHostCaExtension',
    'SshUserCaType',
    'SshUserCaExtension',
)
```

I guess I'm a little confused. Are you saying you want the imports enumerated in both the module and the __init__.py file?

Reply:
@mathiasertl could you clarify this comment:

> Please no wildcard imports (flake8 rightfully complains about these). Also name exported extensions in __all__, as mypy will otherwise complain.

I'm not sure what you want me to change in the `__all__` statement that's already there. Regarding the star import, I'm willing to enumerate the imports in the `__init__` file as well, although I don't understand why the namespace is polluted in this situation. But I'm not disputing your requests; only asking for clarification what you want. 

I'd also be glad if we could speed up the communications tempo.

Reply:
With `from .extensions import ...` (whether wildcard or not), you effectively make extension classes implemented in `django_ca.openssh.extensions` available under the shorter and nicer `django_ca.openssh`. That's all well and good, no complaint about that in general.

You have a similar layout in `django_ca.extensions`. Try disabling one of the members in `__all__` in `django_ca.extensions.__init__` and run mypy against the source code. In this example, I've disabled the `KeyUsage` export:

```
$ mypy --strict ca/django_ca
ca/django_ca/models.py:82: error: Module "django_ca.extensions" does not explicitly export attribute "KeyUsage"; implicit reexport disabled  [attr-defined]
ca/django_ca/forms.py:31: error: Module "django_ca.extensions" does not explicitly export attribute "KeyUsage"; implicit reexport disabled  [attr-defined]
...
```

So that's why the `__all__` should be in `__init__.py`. (I believe pylint will also complain).

When you have a wildcard import, e.g. flake8 will not be able to catch some errors. Star imports are thus itself a flake8 warning, and any variable used from those star imports will also throw one.

Reply:
Thanks for the explanation; I'm following you now, I believe. Does my latest commit match what you want?